tc-package.sh

```diff
@@ -10,11 +10,16 @@ tar -C ${HOME}/DeepSpeech/tf/bazel-bin/tensorflow/ \
 
 tar -C ${HOME}/DeepSpeech/tf/bazel-bin/native_client/ \
 	-uf /tmp/artifacts/native_client.tar \
-	libdeepspeech.so
+	libdeepspeech.so \
+	libdeepspeech_utils.so
 
 tar -C ${HOME}/DeepSpeech/ds/native_client/ \
 	-uf /tmp/artifacts/native_client.tar \
 	deepspeech
 
+if [ -s ${HOME}/DeepSpeech/ds/native_client/dist/* ]; then
```

Comment:
This is a much better idea :) -s only works against a single file, so if multiple files were in dist/, this wouldn't work correctly. At the moment, that isn't the case, but I like your solution better.